binary-codec/definitions/definitions.json

```diff
@@ -2926,56 +3008,59 @@
     "tecARRAY_TOO_LARGE": 191
   },
   "TRANSACTION_TYPES": {
-    "Invalid": -1,
-    "Payment": 0,
-    "EscrowCreate": 1,
-    "EscrowFinish": 2,
+    "AccountDelete": 21,
     "AccountSet": 3,
+    "AMMBid": 39,
+    "AMMCreate": 35,
+    "AMMDelete": 40,
+    "AMMDeposit": 36,
+    "AMMVote": 38,
+    "AMMWithdraw": 37,
+    "CheckCancel": 18,
+    "CheckCash": 17,
+    "CheckCreate": 16,
+    "Clawback": 30,
+    "Contract": 9,
+    "CredentialAccept": 59,
```

Comment:
The 3 Credential (Accept, Create and Delete) are new, the rest is sorted alphabetically.